oar/core/advisory.py

```diff
@@ -1,3 +1,5 @@
+from datetime import datetime
+from dateutil import parser
```

Comment:
I think this module is not python build-in, we need to update install profile https://github.com/openshift/release-tests/blob/master/pyproject.toml